ui/src/views/AutogenView.vue

```diff
@@ -986,6 +986,12 @@ export default {
         }
         this.itemCount = apiItemCount
 
+        if (this.dataView && this.$route.path.includes('/zone/') && 'listVmwareDcs' in this.$store.getters.apis) {
```

Comment:
Does this need to be in autogenview or in zone specific detail view/tab ?

Reply:
we don't have an explicit component to control viewing the specific action. Hence, this updates the resource value before the zone.js  (https://github.com/apache/cloudstack/blob/main/ui/src/config/section/infra/zones.js#L237) file consumes it to determine whether or not to show the specific button.